andela/travel_tool_front
# #162006704 Add responsiveness to requests modal

#### What does this PR do?
- This PR adds mobile responsiveness to the `New Request` Modal.

#### Description of Task to be completed?
- Add CSS styling to make the `New Request` Modal mobile responsive.

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- checkout the branch `git checkout ch-change-badge-for-approved-requests-161975169`
- Start the server with `make start`
- Start the backend of this project using this link [Click here](https://github.com/andela/travel_tool_back)
- Visit https://travela-local.andela.com
- Login to the application
- Navigate to the request page
- Create a new request.
- Change the size of the browser accordingly to simulate mobile devices resolutions.
- The modal should resize according to the display size.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162006704](https://www.pivotaltracker.com/story/show/162006704)

#### Screenshots (if appropriate)
<img width="900" alt="screen shot 2018-11-20 at 15 35 16" src="https://user-images.githubusercontent.com/12511966/48773986-fb1f7000-ecd9-11e8-97ee-c2b64cb62c98.png">
<img width="893" alt="screen shot 2018-11-20 at 15 35 37" src="https://user-images.githubusercontent.com/12511966/48773989-fb1f7000-ecd9-11e8-96f2-c17f6858f731.png">


#### Questions:
N/A
